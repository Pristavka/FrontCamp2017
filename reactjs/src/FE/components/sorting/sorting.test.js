import React from 'react';
import Sorting from './sorting';

it('Should render a Sorting', () => {

  const props = {
    sortPosts: jest.fn(),
    posts: [{
      '_id': 12354347,
      'author': 'Siarhei',
      'text': 'Hello World!'
    }]
  }

  const wrapper = shallow(<Sorting {...props}/>);
  expect(wrapper).toMatchSnapshot();
});

