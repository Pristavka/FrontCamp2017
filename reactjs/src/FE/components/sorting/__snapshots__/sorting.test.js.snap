// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render a Sorting 1`] = `
<button
  className="button"
  onClick={[Function]}
>
  Sort by author
</button>
`;
